financial-connections/src/main/java/com/stripe/android/financialconnections/analytics/FinancialConnectionsInternalEvent.kt

```diff
@@ -0,0 +1,416 @@
+package com.stripe.android.financialconnections.analytics
+
+import com.stripe.android.financialconnections.domain.ConfirmVerification.OTPError
+import com.stripe.android.financialconnections.exception.FinancialConnectionsError
+import com.stripe.android.financialconnections.exception.WebAuthFlowFailedException
+import com.stripe.android.financialconnections.model.FinancialConnectionsSessionManifest.Pane
+import com.stripe.android.financialconnections.utils.filterNotNullValues
+
+/**
+ * Event definitions for Financial Connections.
+ */
+internal sealed class FinancialConnectionsInternalEvent(
```

Comment:
Would a better name for this be `FinancialConnectionsAnalyticsEvent`?

Reply:
there was an old (but still needed) `FinancialConnectionsAnalyticsEvent` that is used to fire 3 legacy events to our old analytics platform. 

Went ahead and made that a private class and renamed `FinancialConnectionsInternalEvent` to `FinancialConnectionsAnalyticsEvent` - https://github.com/stripe/stripe-android/pull/7344/commits/f8b78261cf21e055bd35f8ccaeb411f33a6c2535